demos/tests/cases.py

```diff
@@ -237,8 +240,13 @@ def single_option_cases(key, *args):
             *combine_cases(
                 TestCase(options={'-at': 'yolo'}),
                 single_option_cases('-m',
+                    ModelArg('person-vehicle-bike-detection-crossroad-yolov3-1020'),
                     ModelArg('yolo-v3-tf'),
                     ModelArg('yolo-v3-tiny-tf'))),
+            *combine_cases(
+                TestCase(options={'-at': 'faceboxes'}),
+                single_option_cases('-m',
+                    ModelArg('faceboxes-pytorch'))),
```

Comment:
There are no other `faceboxes` models. You should merge `-at` and `-m` here.

Reply:
ok, changed. also put options in alphabet order

Reply:
Wrapping with `*combine_cases()` is not needed: product of one element is just that element.

Reply:
And as the test says there is no `--architecture_type` for C++ version, only `-at`.

Reply:
fixed

Reply:
There is `combine_cases()` left for python